bench/report/runs.ml

```diff
@@ -100,6 +100,26 @@ let pp_quick_results fmt results =
     "Nothing: %6i    Reached: %6i    Timeout: %6i    Other: %6i    Killed: %6i"
     !nothing !reached !timeout !other !killed
 
+let pp_table fmt results =
+  let nothing = ref 0 in
+  let reached = ref 0 in
+  let timeout = ref 0 in
+  let killed = ref 0 in
+  let other = ref 0 in
+  List.iter
+    (fun result ->
+      match result.Run.res with
+      | Nothing _ -> incr nothing
+      | Reached _ -> incr reached
+      | Timeout _ -> incr timeout
+      | Signaled _ | Stopped _ -> incr killed
+      | Other _ -> incr other )
+    results;
+  Format.fprintf fmt
+    "| Nothing | Reached | Timeout | Other | Killed |@\n\
+     |:-------:|:-------:|:-------:|:-----:|:------:|@\n\
+     | %6i | %6i | %6i | %6i | %6i |" !nothing !reached !timeout !other !killed
```

Comment:
Could you use `Runs.count_nothing results` and the like instead of recomputing them here ?

Also, should we add tables to display also all the clock/utime/stime statistics ? See `runs.mli` to get the list of available functions. :)

Reply:
In `quick_results` it was done with references because we call it after all tasks so I did not want to make it slow but here it probably does not matter. (And actually it probably doesn't matter in `quick_results` either...`

Reply:
> Could you use Runs.count_nothing results and the like instead of recomputing them here ?

Ofc! :)

> Also, should we add tables to display also all the clock/utime/stime statistics ? See runs.mli to get the list of available functions. :)

Good idea! Will add this also :)

Reply:
And for the time statistics, I guess we want to compute them on `reached` tasks only, right?

Reply:
Hum idk, that might skew the results in favour of the tasks we perform very well? What do you think?

Reply:
Well, the timeout tasks are all going to take 30 sec. and are not very interesting for wall clock time. But I don't mind.